src/adapt/layout.js

```diff
@@ -960,32 +944,40 @@ adapt.layout.Column.prototype.layoutFloat = function(nodeContext) {
 	    if (self.vertical) {
 	    	floatHorBox = adapt.geom.rotateBox(floatBox);
 	    }
+        var dir = self.getBoxDir();
+        if (floatHorBox.y1 < self.bottommostFloatTop * dir) {
+            var boxExtent = floatHorBox.y2 - floatHorBox.y1;
+            floatHorBox.y1 = self.bottommostFloatTop * dir;
+            floatHorBox.y2 = floatHorBox.y1 + boxExtent;
+        }
 	    adapt.geom.positionFloat(box, self.bands, floatHorBox, floatSide);
 	    if (self.vertical) {
 	    	floatBox = adapt.geom.unrotateBox(floatHorBox);
 	    }
+		adapt.base.setCSSProperty(element, "position", "absolute");
 	    adapt.base.setCSSProperty(element, "left",
 	    		(floatBox.x1 - self.getLeftEdge() + self.paddingLeft) + "px");
 	    adapt.base.setCSSProperty(element, "top",
 	    		(floatBox.y1 - self.getTopEdge() + self.paddingTop) + "px");
+        if (nodeContext.clearSpacer instanceof Node) {
+            nodeContext.viewNode.parentNode.removeChild(nodeContext.clearSpacer);
```

Comment:
Though I assume that the current code is correct, I think the following code is more natural:
```javascript
nodeContext.clearSpacer.parentNode.removeChild(nodeContext.clearSpacer);
```